query-connector/src/app/constants.ts

```diff
@@ -314,8 +322,37 @@ export const valueSetTypeToClincalServiceTypeMap = {
   medications: ["mrtc"],
   conditions: ["dxtc", "sdtc"],
 };
+/// TODO: Remove the above once ticket #2789 is resolved
 
-export const metadata = {
-  title: "Query Connector",
-  description: "Try out TEFCA with queries for public health use cases.",
-};
+/*
+ * The expected type of a ValueSet concept.
+ */
+export interface Concept {
+  code: string;
+  display: string;
+  include: boolean;
```

Comment:
Is your intent that when we store the valusesets outside the context of a specific query `include` would be `true` for all concepts?

Reply:
Yes, that's currently how we have it set up too. It would be true for the standard queries and then when a user check/unchecks the boxes in the UI, the values would change and be stored for the saved queries.